### ISSUE TITLE
Lack of config.json by default produces a SyntaxError

### ISSUE THREAD

**Issue by MrTimscampi** [jellyfin/jellyfin-web#1031]
**Describe the bug**

Client settings are trying to load a "config.json" file which, by default, doesn't exist. This produces a SyntaxError in the console.

**To Reproduce**
1. Open web client
2. Open browser console

**Expected behavior**
No error should be produced.

**Logs**
```
SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
```

**Screenshots**

**System (please complete the following information):**
 - OS: Arch Linux
 - Browser: All
 - Jellyfin Version: Master

**Additional context**
Renaming `config.default.json` to `config.json` fixes it, but we shouldn't ask the user to do that before using the client without producing an error.

@dkanada What do you think ? Should it fallback to using `config.default.json` if `config.json` doesn't exist ?

**Comment by dkanada** [comment]
> Should it fallback to using config.default.json if config.json doesn't exist?

That seems like a good plan.

**Comment by JustAMan** [comment]
I'm pretty sure that this `<` is a part of error message page. Why aren't we handling the error instead?

**Comment by JustAMan** [comment]
Or, actually, we should be handling the errors in parsing the JSON and storing an empty object instead, maybe with a warning in the console.

Also I just noticed that current webConfig code doesn't handle baseURL being non-`/`, is this okay?

**Comment by dkanada** [comment]
will be fixed with #1223
